calblueprint/dream-mobile
# Creating, updating, and deleting sessions for courses.

Frontend portion for sessions. See backend PR at [https://github.com/calblueprint/dream-rails/pull/31](url)

**Included in this PR:** 
1. Add sessions
2. Delete sessions
3. Update sessions
4. Move to tcomb forms + some basic styling

**To test:** 
1. Create a new course
2. Click "+ Add Session" to add 2 sessions
3.  Save the course
4. Edit the course: delete a session, and update the remaining session
5. Save the course
6. Edit the course and see that everything has been updated

**Coming soon:** 
1. Updating ViewCourse screen (show teacher names and course sessions)
2. More form validation
3. Sessions seed data
4. Styling
5. Moving over other forms to tcomb
